src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react"
+import styled from "styled-components/native"
+
+import { Colors } from "lib/data/colors"
+import { Fonts } from "lib/data/fonts"
+import { InvertedButton } from "../../Buttons"
+
+// hard-coded values for now.
+const saleArtwork = {
+  artwork: {
+    title: "Meteor Shower",
+    date: "2015",
+    artist_names: "Makiko Kudo",
+  },
+  lot_label: "538",
+}
+
+export class ConfirmBid extends React.Component<ConfirmBidProps> {
+  // TODO move to utils?
+  formatAmountCent(amount) {
+    return !isNaN(amount) ? "$" + (amount / 100).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",") : ""
```

Comment:
One pattern that's helpful when dealing with complex return values is to always assign the operation to a variable and then return that, so that one can log or more easily drop a debugger and operate in between:

```javascript
formatAmountCents(amount) {
  const formatted = !isNaN(amount) ? "$" + (amount / 100).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",") : ""
  return formatted
}
```
its also a helpful to add a comment around regex's showing what the output will be so that future devs can unpack at a glance.

Reply:
I'll make this change after merging this PR.